modules/c-blosc2/metadata.json

```diff
@@ -11,6 +11,7 @@
         "github:Blosc/c-blosc2"
     ],
     "versions": [
+        "2.12.0.bcr.1",
```

Comment:
Your version is newer - it should be the last element in this list - at least it has handled this way in all other metadata.json file - e.g. https://github.com/bazelbuild/bazel-central-registry/blob/6cbcb18f8095f8cf8b4fd9051370a011dd35f94c/modules/zlib/metadata.json#L12